client/core/core.go

```diff
@@ -430,6 +433,55 @@ func (dc *dexConnection) findOrder(oid order.OrderID) (tracker *trackedTrade, pr
 	return
 }
 
+func (c *Core) sendCancelOrder(dc *dexConnection, oid order.OrderID, base, quote uint32) (order.Preimage, *order.CancelOrder, []byte, chan struct{}, error) {
+	preImg := newPreimage()
+	co := &order.CancelOrder{
+		P: order.Prefix{
+			AccountID:  dc.acct.ID(),
+			BaseAsset:  base,
+			QuoteAsset: quote,
```

Comment:
The requirement to have base/quote asset IDs for a cancel order is kind of annoying since just `TargetOrderID` is enough for server to figure it out.  I have a stash that modifies the server's `handleCancel` to loop through the market tunnels, but that doesn't change deployed servers...